### PR TITLE
Add tutorial on setting up your local environment

### PR DESCRIPTION
**Pull request by yuvipanda**
Ref https://github.com/2i2c-org/infrastructure/issues/1167

### CODE REVIEW

File: docs/tutorials/setup.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+## Step 2: Setup the python environment
+
+Our deployment scripts are all written in python, so you'll need to have a recent
+version of Python 3 installed. There are innumerable ways to install python on your
+system, so we will not go into them here. Two quick suggestions are to either use
+[miniforge](https://github.com/conda-forge/miniforge) if you are already familiar with
+`conda`, or use `brew install python3` on a Mac.
+
+Once Python is installed, you need to create a virtual environment to install the specific
+libraries we will use. Again this depends on how you installed python, and wether you
```
Comment (sgibson91) [inline docs/tutorials/setup.md:70]:
```suggestion
libraries we will use. Again this depends on how you installed python, and whether you
```

File: docs/tutorials/setup.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+You need to already have permissions to the `two-eye-two-see` project via
+your `2i2c.org` Google Account. If
+
+1. Run `gcloud auth login`, and when the browser window pops up, login with
+   your `2i2c.org` google account.
+2. Run `gcloud auth application-default login` and repeat the process again.
+
+Tada, now you're logged in!
+
+## Step 3: Getting access to any of our kubernetes clusters with `deployer.py`
```
Comment (sgibson91) [inline docs/tutorials/setup.md:96]:
```suggestion
## Step 3: Getting access to any of our kubernetes clusters with `deployer.py`
```

`deployer.py` is called something else now - the `deployer` module/package/library?

**Review by sgibson91 — APPROVED** [review]
Left a few comments/clarifications but LGTM!

File: docs/tutorials/setup.md (hunk 1 above)
Comment (yuvipanda) [reply docs/tutorials/setup.md:70]: haha, looks like this is one I can never ever get right :D

File: docs/tutorials/setup.md (hunk 2 above)
Comment (yuvipanda) [reply docs/tutorials/setup.md:96]:
```suggestion
## Step 3: Getting access to any of our kubernetes clusters with `deployer` module
```

**Comment by yuvipanda** [comment]
Thanks a lot, @sgibson91!

**Comment by yuvipanda** [comment]
Thanks, @choldgraf and @damianavila

**Pull request merged**